### PR TITLE
[WIP] Sync has ability to sync license

### PR DESCRIPTION
**Pull request by danielmarbach**

### CODE REVIEW

**Comment by johnsimons** [comment]
@danielmarbach I would prefer to have an option to pbot to just add license instead of sync, would that be possible

**Comment by johnsimons** [comment]
Btw, pbot is not working anyway

File: src/PBot/SyncOMatic/DefaultTemplateRepo.cs [hunk 1]
```diff
@@ -31,5 +30,22 @@ static DefaultTemplateRepo()
...
+            if (license == null)
+            {
+                return ItemsToSync;
+            }
+
+            var items = new List<SyncItem>(ItemsToSync);
+            var licenseFile = license != "standard" ? $"{license.ToUpper()}.md" : "RPL.md";
+            items.Add(new SyncItem
+            {
+                Target = "LICENSE.md",
```
Comment (johnsimons) [inline src/PBot/SyncOMatic/DefaultTemplateRepo.cs]: the file should be just `LICENSE`, no extension, see https://developer.github.com/v3/licenses/

Comment (adamralph) [inline src/PBot/SyncOMatic/DefaultTemplateRepo.cs]: It's true that `LICENSE` is standard, but it means we'd have to change most of our repos. Casual browsing indicates that we mostly use `LICENSE.md`. AFAIK the GitHub license API also recognises `LICENSE.md` and `LICENSE.txt`.

**Comment by adamralph** [comment]
@danielmarbach this looks good to me.

Have you smoke tested it?

@johnsimons pbot does work but it's a little crippled right now. Most commands seem to be fine, but others don't work, like `pbot help`. 😓

**Comment by danielmarbach** [comment]
@adamralph I haven't smoke tested it because I haven't set it up on my machine yet. Can you help me out here?

**Comment by danielmarbach** [comment]
@johnsimons if the license has to be in every repo it belongs to the repo standards. I was even thinking about making it mandatory.,

**Comment by adamralph** [comment]
@danielmarbach sure, I'll ping you a bit later regarding a smoke test.

**Comment by johnsimons** [comment]
> if the license has to be in every repo it belongs to the repo standards

The issue @danielmarbach, is that those standards are not applied to all repos, eg https://github.com/Particular/Course.Slides, https://github.com/Particular/Serviceinsight, .... so not all repo adheres to those "standards".
So to make this tool work as a helpful tool, I would make it possible to just use it to add the license.
Not saying that it should not also be in the sync mode.

**Comment by danielmarbach** [comment]
@johnsimons it seems there are only a few exceptions to the repo standards. Not sure why we would need a dedicated command for those few repos. I see two possibilities. You use sync as well and partially merge the PR (with the license changes only) or you update those repos by hand.

**Comment by adamralph** [comment]
PBot has been deprecated.

**Pull request closed without merging**